pkg/runtime/runtime.go

```diff
@@ -2103,7 +2113,7 @@ func (a *DaprRuntime) initActors() error {
 		a.namespace, a.appConfig)
 	act := actors.NewActors(a.stateStores[a.actorStateStoreName], a.appChannel, a.grpc.GetGRPCConnection, actorConfig,
 		a.runtimeConfig.CertChain, a.globalConfig.Spec.TracingSpec, a.globalConfig.Spec.Features,
-		a.resiliency, a.actorStateStoreName)
+		a.resiliency, a.actorStateStoreName, a.workflowEngine.AppChannel)
```

Comment:
This was already technically O(1), even with the string prefix comparison. However, I've since updated this to use a map. Map lookups are probably slower in terms of CPU cycles, but it feels a bit more robust.